MANIFEST.in

```diff
@@ -1,7 +1,7 @@
 include *.rst
 recursive-include doc *
 recursive-include examples *
-recursive-include sklearn *.c *.h *.pyx *.pxd *.pxi *.tp
+recursive-include sklearn *.c *.cpp *.h *.pyx *.pxd *.pxi *.tp
```

Comment:
This is required for the `sdist` to contain `*.cpp` files that is not a source file. Specifically, `svm/src/libsvm/svm.cpp`.